### PR TITLE
Create helper function for ec2 create/tag-on-create IAM permissions

### PR DESCRIPTION
**Pull request by olemarkus**
It's quite difficult to get tag-on-create correctly for EC2 because of the createAction rules etc. The policies coming from AWS are also quite different in strictness, while trying to achieve the same thing. This helper function should help us be consistent with how we create these rules.

### CODE REVIEW

File: pkg/model/iam/iam_builder.go [hunk 1]
```diff
@@ -962,48 +1002,15 @@ func AddAWSEBSCSIDriverPermissions(p *Policy, partition string, appendSnapshotPe
...
-				},
-			),
-			Condition: Condition{
-				"StringEquals": map[string]string{
-					"aws:ResourceTag/KubernetesCluster": p.clusterName,
-				},
-			},
+		[]string{
+			fmt.Sprintf("arn:%v:ec2:*:*:volume/*", partition),
+			fmt.Sprintf("arn:%v:ec2:*:*:snapshot/*", partition),
```
Comment (johngmyers) [inline pkg/model/iam/iam_builder.go:1013]: Perhaps some of this partition logic could be pushed down?

Comment (olemarkus) [reply pkg/model/iam/iam_builder.go]: partition should probably be on the policy object itself. but that seems like something for a follow-up.

**Review by johngmyers — APPROVED** [review]

**Pull request merged**